api/descriptions/feeds/feeds.md

```diff
@@ -1 +1 @@
-Augment the Twistlock Intelligence Stream with your own custom threat data.
+Augments the Twistlock Intelligence Stream with custom threat data. The custom threat data informs Twistlock of how to handle threats specific to your app.
```

Comment:
@KICKdesigns Besides vulns in your app (and any internally built components), customers may have their own intelligence or purchase threat data from third parties (e.g. malicious IPs) that they want to us to report on. Maybe phrase the description this way:

```
Augments the Twistlock Intelligence Stream with custom threat data.
Lets you expand the scope of threats and vulnerabilities that Prisma Cloud can detect and report.
```